tests/vision/docs/test_plots.py

```diff
@@ -28,7 +28,7 @@
 def test_plots_on_gpu():
     """If there is GPU available running all the docs plot files. Only makes sure the plots don't crash, and not \
     testing any other display or functionality."""
-    if torch.cuda.is_available():
+    if not torch.cuda.is_available():
```

Comment:
Why change to not? test name is "test plots on gpu"

Reply:
Big mistake, removed from pr